hdfs3/tests/test_dask.py

```diff
@@ -0,0 +1,461 @@
+from __future__ import print_function, division, absolute_import
+
+import json
+import sys
+import pytest
+from contextlib import contextmanager
+
+pytest.importorskip('distributed')
```

Comment:
Comment requested: we normally *do* perform these tests.

Reply:
Yes, they are run on travis. It's important that the test suite run fine with only required dependencies (or which dask/distributed are not).